libnmstate/nm/route.py

```diff
@@ -141,3 +143,26 @@ def _get_default_route_config(gateway, metric, default_table_id, iface_name):
         Route.NEXT_HOP_ADDRESS: gateway,
         Route.METRIC: metric,
     }
+
+
+def add_routes(setting_ip, routes):
+    for route in routes:
+        _add_specfic_route(setting_ip, route)
+
+
+def _add_specfic_route(setting_ip, route):
+    destination, prefix_len = route[Route.DESTINATION].split('/')
+    prefix_len = int(prefix_len)
+    if iplib.is_ipv6_address(destination):
+        family = socket.AF_INET6
+    else:
+        family = socket.AF_INET
+    metric = route.get(Route.METRIC, Route.USE_DEFAULT_METRIC)
+    next_hop = route[Route.NEXT_HOP_ADDRESS]
+    ip_route = nmclient.NM.IPRoute.new(family, destination, prefix_len,
+                                       next_hop, metric)
+    table_id = route.get(Route.TABLE_ID, Route.USE_DEFAULT_ROUTE_TABLE)
+    if table_id != Route.USE_DEFAULT_ROUTE_TABLE:
```

Comment:
Why not just explicitly set the default? At this point the value is correct, be it default or not.

Reply:
Will set explicitly.